Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -84,6 +84,38 @@ open class ChatMessageListVC: _ViewController,
         return (!listView.isLastCellFullyVisible && isMoreContentThanOnePage) || dataSource?.isFirstPageLoaded == false
     }
 
+    /// A button to scroll the collection view to the first unread message.
+    /// Visible when there are unread messages outside of the bounds of the screen.
+    open private(set) lazy var jumpToUnreadMessagesButton: JumpToUnreadMessagesButton = components
+        .jumpToUnreadMessagesButton
+        .init()
+        .withoutAutoresizingMaskConstraints
+
+    /// A Boolean value indicating whether jump to unread messages button is visible.
+    open var showJumpToUnreadMessages: Bool {
```

Comment:
We usually follow this pattern for these kinds of things, good to have consistency here:
```suggestion
    open var isJumpToUnreadMessagesButtonVisible: Bool {
```

`show` as a prefix usually feels like an action and not a question/boolean